modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -123,6 +123,10 @@ const ControlSelection = (selection: EditorSelection, editor: Editor): ControlSe
       return false;
     }
 
+    if (elm.getAttribute('data-mce-resize') === 'true') {
+      return true;
+    }
```

Comment:
I suspect this is a little too general of a feature, for example the body check below is probably important. Maybe limit it to `span` and/or `div` tags?

Reply:
Maybe we also restrict it to non editable elements only since you can't control select anything. But I wonder if these placeholders should really be resizable. You could add two new predicates to dom/NodeType that is `isDiv` and `isSpan` you can then also the the NodeType.isContentEditableFalse to check if it's non editable.

- [ ] Add test that selects a div and a span that has this attribute set and validate that it has resize handles and possibly even try to resize if it we have some code that does that fake drag/dropping.

Reply:
Update: I have confirmed with Fredrik that this is not intentional. The resize should only apply to our placeholder, not any element with the resize attribute.

- [ ] Change to something like `span.data-mce-resize`, similar to the preview object selector

Reply:
I could do `.tox-uploadcare-placeholder` to be extra specific, but I went with more general selector because I thought the core code should be dummy here and not aware of the uploadcare plugin features 🤔

Reply:
That's fair enough, if we're too general it will encourage integrators to enable resize on things that can't be resized.

How about `span.mce-placeholder`, so it's at least somewhat similar to `span.mce-preview-object`